docs/articles/streams/buffersandworkingwithrate.md

```diff
@@ -189,3 +189,32 @@ var driftFlow = Flow.Create<int>()
 ```
 
 Note that all of the elements coming from upstream will go through expand at least once. This means that the output of this flow is going to report a drift of zero if producer is fast enough, or a larger drift otherwise.
+
+### Repeating Previous Values Downstream
```

Comment:
Wanted to add a lengthier article about `RepeatPrevious` in the context of different sources working at decoupled rates, which is why I added this stage in this first place.